### ISSUE TITLE
How to hide a sheet?

### ISSUE THREAD

**Issue by lporras** [cxn03651/write_xlsx#37]
Hey guys,

I'm trying to hide a sheet but when I open the excel file it is still visible, here is my code:

```ruby
require 'write_xlsx'

workbook = WriteXLSX.new('write_xlsx_example.xlsx')

dropdownSheet = workbook.add_worksheet('DropDown Values')
dropdownSheet.write_row(0,0,['open', 'high', 'close'])
dropdownSheet.protect('password')

worksheet = workbook.add_worksheet

worksheet.data_validation('A1:A100',
{
    :validate        => 'list',
    :value           => "'DropDown Values'!A$1:C$1",
    :input_message   => 'Choose the value from the dropdown',
    :error_message   => 'Please use the dropdown selector to choose the value'
})

worksheet.set_first_sheet
dropdownSheet.hide()
workbook.close
```
I also tried to activate the variable 'worksheet' but excel shows the two sheets as active.

**Comment by cxn03651** [comment]
Thanks for your bug report about problem when hide first sheet.

Fixed version 0.85.3 has been released.
please try it.

**Comment by lporras** [comment]
hey @cxn03651 it works now, thanks!
